brickstrap.sh

```diff
@@ -461,14 +517,18 @@ function brp_configure_packages () {
         fi
     done
 
-    # run dpkg `--configure -a` twice because of errors during the first run
+    brp_fixup_etc_shells
+
     info "configuring packages..."
-    br_chroot_bind /usr/bin/dpkg --configure -a || \
-    br_chroot_bind /usr/bin/dpkg --configure -a || true
+    br_chroot_bind /usr/bin/dpkg --configure -a
+
+    export PATH="$BRP_OLD_PATH"
 
     # remove our temporary awk symlink as it is no longer required.
-    info "Removing awk temporary symlink"
-    br_chroot rm -f /usr/local/bin/awk
+    if [ -x "/$(br_chroot_brp_dir)/bin/awk" ]; then
```

Comment:
Since we are removing the symlink here, it seems like it would be simpler to just put it at `/usr/local/bin` instead of `/brickstrap/bin` so that we don't have to mess with the path variable here. If it were sticking around longer, it would make sense to have it under `/brickstrap` so that it was excluded from the tar file, but that is not the case here.